### DISCUSSION TITLE
Permissão do Usuário para Conexão remota

### DISCUSSION THREAD

**Discussion by GabrielXRD** [amidaware/tacticalrmm#1497] [Q&A]
Gostaria de saber se alguem tem alguma solução para que o usuário permita ou não a conexão remota.
Minha duvida é referente a diversas normas de LGPD e me preocupo em implantar o RRM sem ter tal autenticação, como por exemplo o Anydesk oferece, como por exemplo:

Fulano deseja conectar-se:

Permitir.            ----         Não Permitir.

**Comment by silversword411** [comment]
<img width="487" alt="2023-05-01_161539 - chrome_Tactical_RMM_Documentation_-_Google_Chrome" src="https://user-images.githubusercontent.com/13395348/235523413-d2e745d8-6dbe-4a76-aa1a-7091879dbc6f.png">

**Comment by GabrielXRD** [comment]
Thakss!!!
